### ISSUE TITLE
Have a basic linux setup guide 

### ISSUE THREAD

**Issue by shirishag75** [mars-sim/mars-sim#62]
Need to have a basic linux setup guide, see 

https://github.com/Guake/guake as an inspiration. 

You might have more people helping if the onboarding pocess is at least documented.

**Comment by larsnaesbye** [comment]
I agree with this. Should this be in README.md or somewhere else?

**Comment by mokun** [comment]
Yes. We can test on a few popular linux distro such as Fedora, Ubuntu, Debian, etc. and report our findings here. 

We recommend using Oracle's Java 8 u151. Each linux distro has their own way of installing Java JDK/JRE. 

In the past, I've also successfully run mars-sim using OpenJDK together with OpenJFX under Ubuntu.

**Comment by mokun** [comment]
see https://github.com/mars-sim/mars-sim/issues/53 as reference

**Comment by mokun** [comment]
Wow this thread was from over 7 yrs ago.
